README.md

```diff
@@ -1,2 +1,3 @@
 # git-demo
 to demonstrate how to use git and GitHub
+# name: hongtong lin
```

Comment:
Could you please add a `-` so this becomes a list item?

Please do not include a `#`, because that will make a header. 

Also, please make your name into a link that goes to that URL. The URL should not be visible.